package.json

```diff
@@ -154,7 +154,9 @@
         "yargs": "^17.7.2"
     },
     "dependencies": {
+        "@types/d3": "^7.4.3",
```

Comment:
Would be better to move @types under devDependencies, as types don't participate to build

npm install --save-dev @types/d3